### PR TITLE
Add Light theme

### PR DESCRIPTION
**Pull request by DhruvThakker**
#6 Changed palette to a dictionary and added a light theme.

### CODE REVIEW

File: core.py [hunk 1]
```diff
@@ -15,10 +15,11 @@ class ZulipController:
     the application.
     """
 
-    def __init__(self, config_file: str) -> None:
+    def __init__(self, config_file: str, theme:str) -> None:
```
Comment (amanagr) [inline core.py]: `theme: str='default'`
we don't want to make it compulsory to add a theme while running the program.

File: run.py [hunk 2]
```diff
@@ -15,14 +15,17 @@ def parse_args():
...
+                        help='choose color theme. (e.g. light)')
 
     args = parser.parse_args()
     return args
 
 def main():
     args = parse_args()
     try:
-        ZulipController(args.config_file).main()
+        ZulipController(args.config_file,args.theme).main()
```
Comment (amanagr) [inline run.py]: give space after `,`

File: core.py (hunk 1 above)
Comment (DhruvThakker) [reply core.py]: The default is mentioned in arg parser.
Value of theme is 'default' initially.

**Comment by amanagr** [comment]
@DhruvThakker 
Put a space after `:` in `theme:str`.
Squash the commits into a nice commit message.
Post a screen shot of the light theme.
Thanks.

**Comment by DhruvThakker** [comment]
image
Here is the screenshot of a minimalistic light theme.
Requested Changes are done.

**Pull request closed without merging**